jenkins/webhook-proxy/main.go

```diff
@@ -307,6 +316,11 @@ func (s *Server) HandleRoot() http.HandlerFunc {
 
 		log.Println(requestID, event)
 
+		if !event.IsValid() {
```

Comment:
Perhaps isValid could return a short error description which is returned to the caller?

Reply:
You are right, we could improve this. Overall, I wasn't quite sure what to check for. There are also quite a few packages that deal with validation, which might provide additional value, but I wasn't quite ready yet to make the jump from "no dependency" to "one dependency". I'll leave it like this for now, and we can improve this further, but I think that also entails some research what each individual param's values are ...